src/test/java/uk/gov/justice/probation/courtcaseservice/jpa/repository/HearingRepositoryFacadeIntTest.java

```diff
@@ -270,104 +270,6 @@ private static HearingEntity getHearingEntity(HearingDefendantEntity hearingDefe
         return hearingEntity;
     }
 
-    @Test
-    // TODO: is this a possible scenario? Check with John
```

Comment:
Should this be in a ticket `// TODO: is this a possible scenario? Check with John` ? as a spike mayb...

Reply:
Maybe - it is a 2 year old comment made from someone who has left to someone else who has left.
The method where the assertion is failing `hearingEntityInitService.findFirstByHearingIdFullyInitialised` is a test only method anyway

Like I say, I don't really understand what this test is for.

Reply:
An alternative would be to delete the last assertion and leave the previous one
` assertThat(hearingRepositoryFacade.save(hearingEntity)).isEqualTo(hearingEntity);`